lib/calc_route.dart

```diff
@@ -75,7 +75,7 @@ bool check_viable_tack(CartPoint p1, CartPoint p2, double wind_radians) {
   double ngzone_upper_bound = (wind_radians + no_go_limit_radians);
   double ngzone_lower_bound = (wind_radians - no_go_limit_radians);
   verbosePrint("wind lower-upper: " + radToDegStr(ngzone_lower_bound) + " <-> " + radToDegStr(ngzone_upper_bound));
-  Point delta = p2.toPoint() - p1.toPoint();
+  Vector2 delta = p2 - p1;
   double tack_angle = atan2(delta.y, delta.x);
   verbosePrint("pot viable tack: " + radToDegStr(tack_angle));
   // arctan gives values fro [-pi, pi]
```

Comment:
normally arctan would give -pi/2 to pi/2, but since its atan2, the signs of y and x are preserved. Thus the function can determine correct angle from -pi to pi.